dashboard/map.js

```diff
@@ -1,4 +1,4 @@
-mapboxgl.accessToken = 'pk.eyJ1IjoibmluYWRpY2FyYSIsImEiOiJjazNsb3V3Ym0wNTdqM2JvaGRhdjFlN21vIn0.Q4kTCqSpKI00NAJzezecZg';
+mapboxgl.accessToken = 'pk.eyJ1IjoiY21vcmVub3N0b2tvZSIsImEiOiJjazg5MGZ4OHYwMXA5M25wazBtZXA2dGxwIn0.-6W7ECy-ha5nIUXZ-1o8mg';
```

Comment:
I am not sure this change has been included in the log?

Reply:
I needed to change the key to actually test the map following Oliver's change